### DISCUSSION TITLE
Is there a way to format widgets in bar?

### DISCUSSION THREAD

**Discussion by yuandi42** [qtile/qtile#3397] [Q&A] [answered]
![my bar](https://user-images.githubusercontent.com/42729673/156334590-60af881f-6bf9-4272-b569-83e3aa70bb3a.png)
Pic above is my qtile statusbar, in which all widgets crowd in left side. I wonder if I can put different widgets to different side, or set them right, left, centered respectively.

**Comment by elParaguayo** [comment, marked as answer]
Yes - `widget.Spacer` is your friend here ([docs](http://docs.qtile.org/en/stable/manual/ref/widgets.html#spacer)).

If you use one `Spacer` then the widgets before it will be on the left of the bar and the ones after it will be on the right.

If you use two `Spacer`s then the widgets between the `Spacer`s should be in the middle of the bar.
